tests/testthat/_snaps/Design-methods.md

```diff
@@ -1,3 +1,978 @@
+# simulate produces consistent results with placebo data
+
+    Code
+      result
+    Output
+      An object of class "Simulations"
+      Slot "fit":
+      [[1]]
+              middle       lower       upper
+      1  0.002380255 0.000638856 0.007082031
```

Comment:
these numbers are very fragile and can easily change between systems/ seeds etc. - it would be much better to avoid snapshot tests here, but instead look at the structure of the result and expect certain things about it (e.g. the class, names, etc. , not everything but pragmatic subset of features of the results)

Reply:
Let's discuss on your return.

Reply:
As discussed at the last team meeting, we will keep the snapshot-based tests for now, accepting that they are an interim.  After the CRAN release is finalised, we will revisit and convert to more "thoughtful" tests based on explicit recreation of expected results wherever possible.